docs/tools/testing/testing_greentea.md

```diff
@@ -178,6 +314,191 @@ This detects your attached target and drives the test. If you need to rerun the
 
 For an explanation of the arguments used in this command, please run `mbedhtrun --help`.
 
+### Command-line usage
+
+This section highlights a few of the capabilities of the Greentea command-line interface. For a full list of the available options, please run `mbed test --help`.
+
+#### Listing all tests
+
+You can use the `-l` argument to list all available tests:
+
+```
+$ mbed test -l
```

Comment:
This will technically work, but it isn't really the supported interface (I can go into more details if you like, but its a bit messy).

If you want the list of compilable tests, you want `mbed test --compile-list`. (https://github.com/ARMmbed/mbed-cli/blob/master/mbed/mbed.py#L2744)
If you want the list of tests that are compiled and ready to run, you wan `mbed test --run-list` (https://github.com/ARMmbed/mbed-cli/blob/master/mbed/mbed.py#L2746)